.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
```

Comment:
It would probably be helpful to make this name a bit more descriptive if possible.

Reply:
Do you have something in mind? I think the default action name GitHub provides for this is `swift` and it usually is the same name of the file, `swift.yml`.

If there was a special `build.yml` and `release.yml` then we could name them `build` and `release`. Do you think `build` works better here?

Reply:
Because of the current name, this will show as `ci / test (platform=iOS\ Simulator,OS=17.2,name=iPhone\ 15\ Pro\ Max, ResearchKit) (pull_request)` in the settings which is more specific

Reply:
I think having it say something along the lines of **Build / RK Unit Tests** or something close would be fine. 

If we could avoid displaying the destination information here (platform, simulator, etc) that would also be great.

Reply:
These now show up as `build / ResearchKit Unit Tests`:

<img width="922" alt="image" src="https://github.com/ResearchKit/ResearchKit/assets/8621344/344f4a7d-3738-4a08-ba4c-573376aa3179">